doc/source/visualizing/callbacks.rst

```diff
@@ -835,3 +835,20 @@ Overplot the Path of a Ray
    p.annotate_ray(oray)
    p.annotate_ray(ray)
    p.save()
+
+
+Applying filters on the final image
+-----------------------------------
+
+It is also possible to operate on the plotted image directly by using
+one of the fixed resolution buffer filter as described in
+:ref:`frb-filters`.
```

Comment:
Here would be a good place to mention that the user may need to call `p.refresh()` for the plot to be updated.